model/Software/Properties/fileKind.md

```diff
@@ -0,0 +1,18 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# fileKind
+
+## Summary
+
+Describes if a given file is a directory or non-directory kind of file.
+
+## Description
+
+An SPDX file may represent a file on disk or a directory of files.  In the future, this may be extended to other kinds (e.g. network based files).
```

Comment:
file could be in memory not just "disk".... probably want to rephrase to something like: 
"represent a file on disk" to "represent a specific file"